test/proxy-specs.js

```diff
@@ -53,6 +53,43 @@ describe('proxy', () => {
     body.should.eql('http://localhost:4444/wd/hub/session/123');
     j.sessionId.should.equal('123');
   });
+  describe('getUrlForProxy', () => {
+    it('should modify session id, host, and port', async () => {
+      let j = mockProxy({sessionId: '123'});
+      let newUrl = j.getUrlForProxy('http://host.com:1234/wd/hub/session/456/element/200/value');
```

Comment:
It's ok like this, but would just write:
```
mockProxy({sessionId: '123'})
   .getUrlForProxy('http://host.com:1234/wd/hub/session/456/element/200/value')
   .should.eql('http://localhost:4444/wd/hub/session/123/element/200/value'); 
```
or a small test helper wrapping the it method.